Tasks/Common/beanstalkutils/beanstalkutils.ts

```diff
@@ -1,165 +1,170 @@
-/*
-  Copyright 2017-2018 Amazon.com, Inc. and its affiliates. All Rights Reserved.
-  *
-  * Licensed under the MIT License. See the LICENSE accompanying this file
-  * for the specific language governing permissions and limitations under
-  * the License.
-  */
-
-import tl = require('vsts-task-lib/task');
-import path = require('path');
-import fs = require('fs');
-import Q = require('q');
-import archiver = require('archiver');
-import Beanstalk = require('aws-sdk/clients/elasticbeanstalk');
-import S3 = require('aws-sdk/clients/s3');
+/*!
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT
+ */
+
+import archiver = require('archiver')
+import Beanstalk = require('aws-sdk/clients/elasticbeanstalk')
+import S3 = require('aws-sdk/clients/s3')
+import fs = require('fs')
+import path = require('path')
+import Q = require('q')
+import tl = require('vsts-task-lib/task')
 
 export class BeanstalkUtils {
```

Comment:
Not sure if we have any typescript file name conventions here, but why this file name is all lowercase while others are title case ?

Reply:
The short answer is because they were like that, I can fix it for beanstalkutils in this PR and then later we can do a rename PR for all weirdly named things

Reply:
I think you can file a separate CR specifically for fixing these naming issues.